hack/conformance.sh

```diff
@@ -11,8 +11,14 @@ mkdir -p ${ARTIFACTS}
 echo 'Obtaining KUBECONFIG of the development cluster'
 export KUBECONFIG=$(./cluster-up/kubeconfig.sh)
 
+sonubuoy_args="--wait --plugin _out/manifests/release/conformance.yaml"
+
+if [[ ! -z "$SKIP_OUTSIDE_CONN_TESTS" ]]; then
```

Comment:
@alonSadan That won't work as far as you have `set -u` which will fail immediately if some var is unset.

Reply:
it will be a problem only if you run the script not using the Makefile. In the makefile, since we are exporting the var, it will work even if we set it to empty string. 
It says so in the script `# This script should be executed through the makefile via `make conformance`.` , so I assumed it will only run with the makefile. 

shouldn't I?

Reply:
@alonSadan Yes sure, since now it's export is coupled with the script calling.